### PR TITLE
bots: Replace in rhel libvirt package requirement with libvirt-daemon-kvm

### PR DESCRIPTION
**Pull request by KKoukiou**
* [x] image-refresh rhel-x

### CODE REVIEW

**Comment by martinpitt** [comment]
Note that tests won't succeed, as the package currently depend on libvirt. So you need to also change the dependency here, or temporarily add an OR so that the spec works with both the current and the new images.

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-wvqv1.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181002-193939/

**Comment by cockpituous** [comment]
image-refresh rhel-x done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-x-20181002-200808

**Comment by martinpitt** [comment]
Fixed wrong image names, and retrying.

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-qz5rr.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181004-060404/

**Comment by cockpituous** [comment]
image-refresh rhel-7-5 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-7-5-20181004-064242

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-z753z.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181004-063939/

**Comment by cockpituous** [comment]
image-refresh rhel-7-6 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-7-6-20181004-071313

**Comment by martinpitt** [comment]
Rebuilding centos-7 too.

**Comment by cockpituous** [comment]
image-refresh in progress on centosci-tasks-2z007.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181004-074545/

**Comment by cockpituous** [comment]
image-refresh centos-7 done: https://github.com/cockpituous/cockpit/commits/image-refresh-centos-7-20181004-080404

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-wtg2g.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181004-105959/

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-4v0t0.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181004-113838/

**Comment by cockpituous** [comment]
image-refresh rhel-7-5 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-7-5-20181004-121212

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-rcm9f.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181004-121212/

File: bots/images/scripts/rhel.setup [hunk 1]
```diff
@@ -160,6 +159,10 @@ if [ "$IMAGE" = "centos-7" ]; then
     COCKPIT_DEPS="${COCKPIT_DEPS/redhat-logos/}"
     COCKPIT_DEPS="${COCKPIT_DEPS/atomic-openshift-clients /}"
     COCKPIT_DEPS="$COCKPIT_DEPS kubernetes-client"
+    COCKPIT_DEPS="$COCKPIT_DEPS libvirt"
+fi
+if [ "$IMAGE" == "rhel-7.5" ] || [ "$IMAGE" == "rhel-7.6"] ; then
+    COCKPIT_DEPS="$COCKPIT_DEPS libvirt"
```
Comment (martinpitt) [inline bots/images/scripts/rhel.setup]: centos-7 too

File: bots/images/scripts/rhel.setup [hunk 2]
```diff
@@ -170,6 +173,7 @@ if [ "$IMAGE" = "rhel-x" ]; then
     COCKPIT_DEPS="${COCKPIT_DEPS/atomic /}"
     # docker is not a rhel-x thing (https://bugzilla.redhat.com/show_bug.cgi?id=1602951)
     COCKPIT_DEPS="${COCKPIT_DEPS/docker /}"
+    COCKPIT_DEPS="${COCKPIT_DEPS/libvirt-daemon-kvm /}"
```
Comment (martinpitt) [inline bots/images/scripts/rhel.setup]: I think you mean to *add* the package here, not remove it?

**Comment by martinpitt** [comment]
I removed the tasks for rebuilding rhel/centos-7. This change only affects the rhel-x image.

File: bots/images/scripts/rhel.setup (hunk 1 above)
Comment (KKoukiou) [reply bots/images/scripts/rhel.setup]: Done
Also fixed inconsistent spacing.

File: bots/images/scripts/rhel.setup (hunk 2 above)
Comment (KKoukiou) [reply bots/images/scripts/rhel.setup]: Right and also added libvirt-daemon-config-network which brings default network.

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-6c5bs.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10206-20181005-074444/

**Comment by cockpituous** [comment]
image-refresh rhel-x done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-x-20181005-081212

**Pull request merged**